mmeds/spawn.py

```diff
@@ -66,6 +66,8 @@ def __init__(self, address=("", sec.WATCHER_PORT), authkey=sec.AUTH_KEY):
         self.logger = Logger
         self.current_upload = None
         self.checked_stats = None
+        self.cleaned_temp = datetime.utcnow()
```

Comment:
By setting it this way, the watcher won't do a clean until a day after it's first started. I think it might be better to have it automatically do a clean when it first starts, or within an hour if that makes startup take noticeably longer

Reply:
I can look into doing it on start-up